frontend/src/metabase/query_builder/actions/object-detail.ts

```diff
@@ -148,11 +176,21 @@ export const loadObjectDetailFKReferences = createThunkAction(
       // skipping that for now because it's easier to just run this each time
 
       // run a query on FK origin table where FK origin field = objectDetailIdValue
-      const fkReferences = {};
+      const fkReferences: Record<FieldId | string, FKInfo | undefined> = {};
       for (let i = 0; i < tableForeignKeys.length; i++) {
         const fk = tableForeignKeys[i];
         const info = await getFKCount(card, fk);
-        fkReferences[fk.origin.id] = info;
+
+        if (fk.origin) {
+          const key =
+            typeof fk.origin.id === "number"
```

Comment:
@ranquild am I right, we should handle both cases here: when `origin.id` is a number and when it's a `FieldReference`?

Reply:
you can use `origin_id` - the type is correct here